From b72c04385945c56a9b664e62569d0387ceaa3a5a Mon Sep 17 00:00:00 2001
From: Fathi Boudra <fathi.boudra@linaro.org>
Date: Thu, 12 Oct 2017 10:40:18 +0300
Subject: [PATCH 1/1] commands: nm: fix typo INSTALL_TARTGETS ->
 INSTALL_TARGETS

Fix the typo preventing to install the files.

cp: cannot stat '/opt/ltp/testcases/data/nm01/lib.a': No such file or
directory
1 TBROK: cp /opt/ltp/testcases/data/nm01/lib.a . failed

Signed-off-by: Fathi Boudra <fathi.boudra@linaro.org>

Upstream-Status: Backport [https://github.com/linux-test-project/ltp/commit/6570df2d8cc894da6309acdca84dddc65a87851f]
---
 testcases/commands/nm/datafiles/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/commands/nm/datafiles/Makefile b/testcases/commands/nm/datafiles/Makefile
index 3514665a9..371490e1c 100644
--- a/testcases/commands/nm/datafiles/Makefile
+++ b/testcases/commands/nm/datafiles/Makefile
@@ -15,7 +15,7 @@ top_srcdir		?= ../../../..
 
 include $(top_srcdir)/include/mk/env_pre.mk
 INSTALL_DIR		:= testcases/data/nm01
-INSTALL_TARTGETS	:= f1 f2 f3 lib.a
+INSTALL_TARGETS		:= f1 f2 f3 lib.a
 MAKE_TARGETS		+= f1 f2 f3 lib.a
 
 lib.a: f1.o f2.o f3.o
-- 
2.14.2

